rust/garbage_collector/tests/prop_test_local_files.rs

```diff
@@ -760,7 +767,6 @@ impl GcTest {
         self.last_cleanup_files = Vec::new();
         match orchestrator.run(system.clone()).await {
             Ok(response) => {
-                tokio::time::sleep(tokio::time::Duration::from_secs(2)).await;
```

Comment:
this sleep was causing the test to run for > 1m
the test runs in < 3s without it